src/browser.rs

```diff
@@ -226,6 +228,46 @@ impl Browser {
             .await?;
         Ok(rx.await?)
     }
+
+    /// Create Listener for Target Changed Event
+    pub async fn target_changed_listener(&self) -> Result<EventStream<EventTargetInfoChanged>> {
+        let (tx, rx) = unbounded();
+        self.sender
+            .clone()
+            .send(HandlerMessage::AddEventListener(
+                EventListenerRequest::new::<EventTargetInfoChanged>(tx),
+            ))
+            .await?;
+
+        Ok(EventStream::new(rx))
+    }
+
+    /// Create Listener for Target Created Event
+    pub async fn target_created_listener(&self) -> Result<EventStream<EventTargetCreated>> {
+        let (tx, rx) = unbounded();
+        self.sender
+            .clone()
+            .send(HandlerMessage::AddEventListener(
+                EventListenerRequest::new::<EventTargetCreated>(tx),
+            ))
+            .await?;
+
+        Ok(EventStream::new(rx))
+    }
+
+    /// Return page of given target_id
+    pub async fn get_page(&self, target_id: TargetId) -> Result<Page> {
```

Comment:
This is an essential addition to the browser API.

Reply:
Could you please split this to another PR, since this is unrelated to event handling and can be merged asap.

Reply:
Okay, I split it.  #38.